spec/presenters/register_api/single_application_presenter_spec.rb

```diff
@@ -184,8 +184,8 @@
     it 'compacts two nationalities with the same ISO value' do
       application_form = create(:application_form,
                                 :minimum_info,
-                                first_nationality: 'Welsh',
-                                second_nationality: 'Scottish')
+                                first_nationality: 'British',
+                                second_nationality: 'British')
```

Comment:
Welsh and Scottish are not selection options -- there are no application forms where Welsh or Scottish are recorded as nationalities in the past 3 years.